webapp/api/blog.py

```diff
@@ -1,9 +1,12 @@
-from webapp.api import cache
+from webapp.api import requests as api_requests
```

Comment:
Couldn't you do `from webapp import api` and then refer to `api.requests`? This would achieve the unique naming you're looking for without fundamentally renaming the function.